### DISCUSSION TITLE
Not generated records /libs

### DISCUSSION THREAD

**Discussion by jaroslavbarta** [Netcentric/accesscontroltool#702] [General]
We generate user groups in AEM cloud via NeCentric tool. We are facing issue that records starting with "/libs" in YAM files are not created within ACEs on Author servers. For example:

path: /libs/granite/ui/content/shell/start/content/cq/items/tabs/items/tools
         permission: deny
         privileges: jcr:read

is ignored. 
AEM version: 2024.4.15977.20240418T174835Z. Have you got experience with that?

Thank you,

Jaroslav

**Comment by kwin** [comment]
In general this works. Please share/inspect logs to see what the underlying issue is: https://github.com/Netcentric/accesscontroltool/blob/develop/docs/ApplyConfig.md#startup-hook.

**Comment by jaroslavbarta** [comment]
We have recieved information from Adobe that /libc records are created during deployment only.

**Reply by kwin** [reply]
Yes, that is when ACTool generates the ACEs
